lib/commons/color/get-background-color.js

```diff
@@ -207,17 +207,22 @@ color.getRectStack = function(elm) {
 function sortPageBackground(elmStack) {
 	let bodyIndex = elmStack.indexOf(document.body);
 	let bgNodes = elmStack;
+	// prettier-ignore
+	let sortBodyElement =
+		bodyIndex > 1 || // only if there are negative z-index elements
+		bodyIndex === -1; // only when an element is positioned outside of the body's rect
 
 	if (
-		// Check that the body background is the page's background
-		bodyIndex > 1 && // only if there are negative z-index elements
+		sortBodyElement &&
 		!color.elementHasImage(document.documentElement) &&
 		color.getOwnBackgroundColor(
 			window.getComputedStyle(document.documentElement)
 		).alpha === 0
 	) {
-		// Remove body and html from it's current place
-		bgNodes.splice(bodyIndex, 1);
+		// Only remove document.body if it was originally contained within the element stack
+		if (bodyIndex !== -1) {
```

Comment:
OK, finally figured out what's going on :P. Let's flip around this if so it's not a double negative, maybe add in one or two more comments to explain it.

```js
if (bodyIndex > 1) { // If the body is in the stack, take it out
```